### PR TITLE
Feat/css styling

### PR DESCRIPTION
**Pull request by tenthwalker**
# Description
This pull request includes commits that initialize the CSS styling, connect the CSS/HTML/JS files, and update HTML ids to be more flexible with styling.

# Contributors
@tenthwalker

# Notes
CSS needs the server running to actually be visible.

# Checklist
- [x] My PR has an appropriately descriptive and concise title.
- [x] My PR denotes any/all team members who contributed to it.
- [x] My code follows the Turing Style Guides and best practices.
- [x] I ran the code locally and verified that there are no visible errors.
- [x] feat: My PR clearly describes what feature I'm adding and any changes needed to make it work.

### CODE REVIEW

**Review by BrendanTurner1 — APPROVED** [review]
Looks good. We can work on figuring out what tags we want to use to display the information from the User object.

**Review by ericahagle — CHANGES_REQUESTED** [review]

File: dist/index.html [hunk 1]
```diff
@@ -3,20 +3,20 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- <link rel="preconnect" href="https://fonts.googleapis.com">
```
Comment (ericahagle) [inline dist/index.html:6]: Looks like you need to un-comment these, too, right?

**Review by ericahagle — COMMENTED** [review]
@tenthwalker Looks like it's all working now (Huzzah! 🎉 ) but the styling looks funky on my screen. Are you planning to create another ticket and PR to get the CSS to fully look the way you want? Or do you want to finish it in this PR before we merge?

**Comment by ericahagle** [comment]
> @tenthwalker Looks like it's all working now (Huzzah! 🎉 ) but the styling looks funky on my screen. Are you planning to create another ticket and PR to get the CSS to fully look the way you want? Or do you want to finish it in this PR before we merge?

Screenshot 2023-10-16 at 2 37 48 PM
Screenshot 2023-10-16 at 2 37 55 PM

**Comment by tenthwalker** [comment]
you can merge it in, I have more to do still but I'm gonna open up another branch to do so @ericahagle

**Review by ericahagle — APPROVED** [review]
Okiee dokiee! I'll go ahead and approve this PR as the baseline CSS, and expect a future PR with an updated version. 👍

**Pull request merged**